src/main/java/com/hubspot/ringleader/watcher/WatcherFactory.java

```diff
@@ -160,7 +160,7 @@ public void stateChanged(CuratorFramework client, ConnectionState newState) {
           switch (newState) {
             case SUSPENDED:
             case LOST:
-              LOG.error("Connection lost or suspended, replacing client");
+              LOG.info("Connection lost or suspended, replacing client");
```

Comment:
This is adjacent to this change, but should these statements say more about what client / curator is being operated on?